app/javascript/managers.js

```diff
@@ -11,21 +11,25 @@ export function canInstallUserJS() {
 }
 
 export async function canInstallUserCSS() {
-  if (localStorage.getItem('stylusDetected') == '1') {
+  if (localStorage.getItem('stylusDetected') === '1') {
     return true;
   }
-  postMessage({ type: 'style-version-query', name: "whatever", namespace: "whatever", url: location.href }, location.origin);
-  return new Promise(resolve => setTimeout(function() {
-    resolve(localStorage.getItem('stylusDetected') == '1')
-  }, 200));
+  await new Promise(resolve => {
```

Comment:
Took me a few tries to figure out what's going on here, which probably means there should be a comment.